io.github.wereturtle.ghostwriter.json

```diff
@@ -0,0 +1,120 @@
+{
+    "app-id": "io.github.wereturtle.ghostwriter",
+    "base": "io.qt.qtwebkit.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "ghostwriter",
+    "rename-icon": "ghostwriter",
+    "rename-desktop-file": "ghostwriter.desktop",
```

Comment:
Ideally these should be correctly named in the project repo instead.

Reply:
Names don't matter though for the tooling that extracts info or display icons. It just ensures there is no conflict in the future and does not change anything for downstream packagers.

Reply:
OK, I will keep that in mind for the next repo version.